<launch>
    <node name="star3single" pkg = "iap3spherical" type = "star3single" output = "screen">
        <param name = "urdfPath" type = "str" value = "\\home\\siris\\catkin_ws\\src\\raisim_iap_3spherical\\rsc\\ava\\urdf\\star3.urdf" />
        <param name = "robotName" type = "str" value = "" />
        <param name = "localHostIp" type = "str" value = "172.19.221.5" />
        <param name = "unrealIp" type = "str" value = "172.19.208.1" />
        <param name = "dataPort" type = "str" value = "7050" />
        <param name = "raisimServerPort" type = "int" value = "8080" />
        <param name = "rosbagPath" type = "str" value = "/home/siris/catkin_ws/rosbags/star_replay-20240323.bag" />
    </node>
</launch>